### DISCUSSION TITLE
Detection Confidence Score

### DISCUSSION THREAD

**Discussion by fcuenca4** [esimov/pigo#64] [Q&A] [answered]
Hi! first of all, thanks for all the effort put on this project. 

My question is fairly simple: is there any way to adapt the scoring algorithm so it results in a value `0 < X < 1`? 
I'm curious if this can be done by calculating the max value of the trees. 

Thanks,
Fran

**Comment by esimov** [comment, marked as answer]
The scoring information is encoded into the detection binary tree, so it's not really possible to adapt it. What you can do instead is to check if the detection score is below or above a certain threshold. The detection score is more accurate if it reach above a certain threshold (~0.995).
